lib/content-services/src/lib/viewer/components/alfresco-viewer.component.spec.ts

```diff
@@ -292,17 +297,93 @@ describe('AlfrescoViewerComponent', () => {
         expect(component['nodesApi'].getNode).toHaveBeenCalledTimes(2);
     }));
 
-    it('should not setup the node twice if the node id is not changed', fakeAsync(() => {
-        spyOn(component['nodesApi'], 'getNode').and.stub();
+    it('should not setup the node twice if both nodeId and versionId remain the same', fakeAsync(() => {
+        spyOn(component['nodesApi'], 'getNode').and.returnValue(
+            Promise.resolve(new NodeEntry({ entry: new Node({ name: 'file1', content: new ContentInfo() }) }))
+        );
+        spyOn(component['versionsApi'], 'getVersion').and.returnValue(
```

Comment:
I know that in existing test there was already code like component['nodesApi'] but it's incorrect. We should not refer to private things in tests. Instead I see there are public getters so we should spy on those getters